tools/markdown-magic/test/include-code.md

```diff
@@ -0,0 +1,53 @@
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE_CODE:path=./sample-code.ts) -->
+
+<!-- prettier-ignore-start -->
+<!-- NOTE: This section is automatically generated by embedding the referenced file contents. Do not update these generated contents directly. -->
+
+```
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export function foo(): number {
+	return 42;
+}
+```
+
+<!-- prettier-ignore-end -->
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE_CODE:path=./sample-code.ts&language=typescript) -->
+
+<!-- prettier-ignore-start -->
+<!-- NOTE: This section is automatically generated by embedding the referenced file contents. Do not update these generated contents directly. -->
+
+```typescript
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export function foo(): number {
+	return 42;
+}
+```
+
+<!-- prettier-ignore-end -->
+
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE_CODE:path=./sample-code.ts&language=typescript&start=6&end=-2) -->
+
+<!-- prettier-ignore-start -->
+<!-- NOTE: This section is automatically generated by embedding the referenced file contents. Do not update these generated contents directly. -->
+
+```typescript
+return 42;
+```
```

Comment:
Did you expect this code block to show up here? Seems out of place.

Reply:
Yeah - the `INCLUDE_CODE` pragma will always introduce a code block. And the return statement was the line I was aiming for with `start=6&end=-2`